release_notes_generator/model/issue_record.py

```
@@ -0,0 +1,290 @@
+
+    def get_commit(self, pr_number: int, commit_sha: str) -> Optional[Commit]:
+        """
+        Gets the commit by its number.
+        Parameters:
+             pr_number (int): The number of the pull request.
+             commit_sha (str): The commit sha.
+        Returns:
+            Optional[Commit]: The commit or None if not found.
+        """
+        if pr_number in self._pull_requests:
+            if commit_sha in self._commits[pr_number]:
```

Ruff (0.12.2)

Use a single `if` statement instead of nested `if` statements

(SIM102)